sections/image-with-text.liquid

```diff
@@ -14,7 +14,9 @@
   }
 {%- endstyle -%}
 
-<div class="image-with-text image-with-text--{{ section.settings.content_layout }} page-width isolate{% if settings.text_boxes_border_thickness > 0 and settings.text_boxes_border_opacity > 0 and settings.media_border_thickness > 0 and settings.media_border_opacity > 0 %} collapse-borders{% endif %}{% unless section.settings.color_scheme == 'background-1' and settings.media_border_thickness > 0 and settings.text_boxes_shadow_opacity == 0 and settings.text_boxes_border_thickness == 0 or settings.text_boxes_border_opacity == 0 %} collapse-corners{% endunless %} section-{{ section.id }}-padding{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
+{% assign scheme1 = settings.color_schemes | first %}
```

Comment:
Here this is an added safety if the schemes were to be re ordered ? 

I don't think we really need this 🤔 if that's the only reason. I don't think we have really ever re ordered our color scheme 🤷 

Though the extra safety isn't a big addition.

Reply:
We couldn't reorder schemes before. At least not in a way that made any difference, because we default to background-1, regardless of anything else. That was never _not_ a background-1 scheme. Now reordering, changing the name, or removing the `background-1` scheme entirely changes the meaning of "scheme 1 is used as a fallback". It's still true in that whatever the first scheme defined in settings_data is used as the fallback, but if that's not background-1 then any logic like this that's counting on that will break.